boards/arm/stm32f411e_disco/doc/index.rst

```diff
@@ -83,6 +83,8 @@ hardware features:
 +-----------+------------+-------------------------------------+
 | GPIO      | on-chip    | gpio                                |
 +-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                |
```

Comment:
add a space to get that last column border to align on the right

Reply:
Uh, sorry.